accessibility/index.html

```diff
@@ -0,0 +1,556 @@
+                    <p><span class="gdi"><strong>Roles</strong></span> define the <strong>type</strong> of element or widget presented to the accessibiltiy API</p>
+                    <p>For example you can give a div a role="link" or role="button". The div will now be announced as a link or a button by a screen reader.</p>
+				</section>
+
+				<!-- ARIA ROLES START -->
+
+				<!-- Exercise 05 START -->
+
+				<section>
+					<h3>Exercise 5</h3>
+						<p align="center">1. Assign a varity of <strong>roles</strong> to a div</p>
+						<p align="center">2. Listen to changes in the screen reader</p>
+						<small><a href="https://jsbin.com/ranoxugeqo/edit?html,output" target="_blank">Try It</a></small>
+				</section>
+
+				<!-- Exercise 05 END -->
+        
+                <!-- More Information on ARIA Roles Start -->
+        
+                <section>
+					<h2>More Information on <span class="gdi">ARIA Roles</span></h2>
+        <small><a href="http://www.w3.org/TR/wai-aria/roles" target="_blank">The Roles Model</a></small>
+				</section>
+    
+                <!-- More Information on ARIA Roles End -->
+
+
+				<!-- ARIA STATES AND PROPERTIES START -->
+				
+				<section>
+                    <h2>ARIA <span class="gdi">States</span> and <span class="gdi">Properities</span></h2>
+					<p>States and Properities announce the state of an element to the accessibility API. Often times, but not always, states are activated by user interaction</p>
+                    <p>For example you can let a user know that a form input field is required by using aria-required="true"</p>
+				</section>
+
+				<!-- ARIA STATES AND PROPERTIES START -->
+
+				<!-- Exercise 06 START -->
+
+				<section>
+					<h3>Exercise 6</h3>
+						<p align="center">1. Assign selected <strong>ARIA States</strong> and <strong>Properties</strong> to Semantic HTML</p>
+						<p align="center">2. Listen to changes in the screen reader</p>
+						<small><a href="https://jsbin.com/pajaputila/1/edit?html,output" target="_blank">Try It</a></small>
+				</section>
+
+				<!-- Exercise 06 END -->
+        
+                <!-- More Information on ARIA States Start -->
+        
+                <section>
+					<h2>More Information on <span class="gdi">ARIA States</span></h2>
+        <small><a href="http://www.w3.org/TR/wai-aria/states_and_properties" target="_blank">ARIA States and Properties</a></small>
+				</section>
+    
+                <!-- More Information on ARIA States End -->
+				
+
+				<!-- Final Project START -->
+
+				<section>
+					<h3>Final Project</h3>
+						<small><a href="http://www.followthebrush.me/workshop/final-exercise-before.html">Project Sitelet</a></small>
+                    <br/>
+                        <small>username: gdi<br/>password: pass123</small>
+                        <small>Completing final project. Recieved permission to user content</small>               
+				</section>
+
+				<!-- Final Project END -->
+
```

Comment:
It would be nice to add a conclusion slide! Talk about outstanding questions – how can students dive deeper into accessibility? What wasn't covered in the workshop? Encourage everybody to keep learning.